### PR TITLE
Mice can pilot spiderbots

### PR DESCRIPTION
**Pull request by abernd**
[tweak]

:cl:
 * tweak: Nanotrasen-brand spiderbot interfaces are now compatible with mice.

### CODE REVIEW

**Comment by BomberBro** [comment]
Will the mouse always show up as brown when added to the spiderbot?

**Comment by abernd** [comment]
@BomberBro 
image

**Comment by Kharkov72** [comment]
now we need a deathsquad mouse spiderbot for the mouse ops

**Comment by abernd** [comment]
@Kharkov72 I thought about adding one with microlasers on it. Maybe in another PR.

**Comment by GreatBigFailure** [comment]
That's wonderful

File: code/modules/mob/living/simple_animal/friendly/spiderbot.dm [hunk 1]
```diff
@@ -327,6 +348,8 @@
 	return SLIP_HAS_MAGBOOTS
 
 /mob/living/simple_animal/spiderbot/say(var/message)
+	if(mouse)
```
Comment (DamianX) [inline code/modules/mob/living/simple_animal/friendly/spiderbot.dm]: This looks out of place, it should happen when the mouse enters/leaves

**Review by DamianX — APPROVED** [review]

**Pull request merged**